blueocean-personalization/src/main/js/components/DashboardCards.jsx

```diff
@@ -7,34 +7,14 @@ import { connect } from 'react-redux';
 import { createSelector } from 'reselect';
```

Comment:
Pushed a lot of logic from this component down into `PipelineCard`. Instead of this component picking out the data and passing a ton of props down into the card, it made sense to just pass the object down and let `PipelineCard` do that work.